### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by thomcatdotrocks**
Updating instructions for building the dev environment.

Signed-off-by: Thom Cleary <me@thomcat.rocks>

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -16,3 +16,6 @@ include:
 
 * Alexandr Korsak (https://github.com/oivoodoo)
   Improved multiple file uploading
+
+*Thom Cleary (https://github.com/thomcatdotrocks)
```
Comment (t3chguy) [inline AUTHORS.rst]: looks like it needs a space

**Comment by t3chguy** [comment]
For setting up a dev environment you still will want to yarn link and whatnot so you can edit those libraries outside of `node_modules`

For making a build (not a dev environment) you're right you don't need the rest.

**Comment by thomcatdotrocks** [comment]
Ahh, okay.  So if I just want to run a version of the `develop` build with the latest commits, I _don't_ have to build the entire dev environment, I only have to `git clone` riot-web and checkout the `develop` branch?

**Comment by t3chguy** [comment]
> Ahh, okay. So if I just want to run a version of the develop build with the latest commits, I don't have to build the entire dev environment, I only have to git clone riot-web and checkout the develop branch?


Yup as per https://github.com/vector-im/riot-web#building-from-source which only `recommends` you set up a dev environment as you may wish to tweak things at that point

If you want you can pull tarballs form buildkite.

https://buildkite.com/matrix-dot-org/riot-web then find the latest green build, go to package > artifacts and download. e.g https://buildkite.com/matrix-dot-org/riot-web/builds/2106#d760ceae-48b9-463c-8afd-5c75d2c4ff57

**Comment by thomcatdotrocks** [comment]
Very good, thank you.  I will close this PR.

**Pull request closed without merging**